packages/react-core/src/components/Chip/examples/Chip.md

```diff
@@ -2,13 +2,32 @@
 id: Chip
 section: components
 cssPrefix: pf-c-chip
-propComponents: ['Chip']
+propComponents: ['Chip', 'ChipGroup']
 ouia: true
 ---
 
-## Examples
+## Chip examples
```

Comment:
```suggestion
## Examples
```

Since the rest of our component docs call this section "Examples", I think it would be best to remain aligned with that - even if we're combining a couple of former components. This also aligns with what we do with [toolbar](https://patternfly-react-pr-8968.surge.sh/components/toolbar), for example.

Reply:
@edonehoo The reason we did this was to differentiate between the two sets of examples in the page - so we have Chip examples and Chip group examples. We used that same pattern when combining Alert and Alert group: https://patternfly-react-pr-8968.surge.sh/components/alert Or are you saying that we don't need a second h2 heading for Chip group examples?

Reply:
Ah, I haven't seen the Alert PR yet. Thanks for linking -- it's helpful to see an example with more "sub" examples in each section. Toolbar is kind of a different case, so it's not the best comparison.

I _was_ saying that we don't need a second h2 heading to split/differentiate the "base" vs "group" examples, because it felt like we were still keeping the two original components separate, just on a new page. But, maybe keeping them split makes sense since they are different React components? 

But this is kind of splitting hairs and I don't have a huge preference either way - so long as we are consistent and make sure that the "blurb" description is updated to cover both example groups. If we do decide to keep them split I would change the headings to "Examples" and "Chip group examples". I still feel inclined to keep the first heading consistent with the rest of our components.